billy-portugal/src/test/java/com/premiumminds/billy/portugal/test/services/export/TestPTCreditNotePDFTransformer.java

```diff
@@ -160,11 +160,10 @@ private PTCreditNoteEntity generatePTCreditNote(PaymentMechanism paymentMechanis
 
         this.createSeries(reference, "AC");
 
-        PTCreditNoteEntity creditNote = null;
-        creditNote = (PTCreditNoteEntity) services.issueDocument(
+        PTCreditNoteEntity creditNote = (PTCreditNoteEntity) services.issueDocument(
             new PTCreditNoteTestUtil(PTAbstractTest.injector).getCreditNoteBuilder(reference), params);
```

Comment:
In order to make the code more readable, i'd suggest seperating each issueDocument parameter per line